### ISSUE TITLE
Release badge is not updating when tagging a new release

### ISSUE THREAD

**Issue by KaneCheshire** [theappbusiness/MasterFastfile#42]
The release badge appears to be showing the wrong release. It's a few behind now.

**Comment by NikolaKirev** [comment]
Looks like our tag names are not formatted properly.
I believe they should start with a `v`.

We also need to make the shield image url dynamic.

Will investigate and will draft a `v3.0.1` release to validate my assumptions.

**Comment by NikolaKirev** [comment]
Ok. It's not specifically the `v` prefix but it's something to do with the way we create releases because usually there is a "Latest release" badge on the newest one and the `/releases/latest` URL points to it. In our case this is not working.

**Comment by NikolaKirev** [comment]
Looking like it's all sorted now. Badge value is evaluated based on the latest release and it's linking to the the latest release tag.
It's worth mentioning that to make the `"Latest release"` label showing up I added the tag and created the release using the GitHub Web UI.

It might be worth figuring out if this is the way to do it or what the difference is to the way we were doing it before.

Closing this issue.
